### PR TITLE
Fidelis - new commands

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
In Progress
## Related Issues
fixes: https://github.com/demisto/etc/issues/17179
https://github.com/demisto/etc/issues/18084
https://github.com/demisto/etc/issues/17293

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Dependencies
Mention the dependencies of the entity you changed as given from the precommit hooks in checkboxes, and tick after tested them.
- [ ] Dependency 1
- [ ] Dependency 2
- [ ] Dependency 3

### CODE REVIEW

**Comment by Shellyber** [comment]
@kirbles19 Please review the yml file and the release notes

File: Integrations/FidelisElevateNetwork/FidelisElevateNetwork.py [hunk 1]
```diff
@@ -413,6 +529,111 @@ def upload_pcap_command():
     })
 
 
+def list_metadata_request(time_frame=None, start_time=None, end_time=None, client_ip=None, server_ip=None,
```
Comment (yaakovi) [inline Integrations/FidelisElevateNetwork/FidelisElevateNetwork.py]: move those function below the `upload_pcap` function (to keep relative functions together)

File: Integrations/FidelisElevateNetwork/FidelisElevateNetwork.py [hunk 2]
```diff
@@ -413,6 +529,111 @@ def upload_pcap_command():
...
+            'MalwareSeverity': event.get('MalwareSeverity'),
+            'PcapFilename': event.get('PcapFilename'),
+            'PcapTimestamp': event.get('PcapTimestamp')
+
+        })
+    context = {
+        'Fidelis.Metadata(val.ID && val.ID == obj.ID)': event_context
+    }
+
+    demisto.results({
```
Comment (yaakovi) [inline Integrations/FidelisElevateNetwork/FidelisElevateNetwork.py]: `return_outputs`

File: Integrations/FidelisElevateNetwork/FidelisElevateNetwork.py [hunk 3]
```diff
@@ -41,6 +42,7 @@ def http_request(method, url_suffix, params=None, data=None, files=None, is_json
         files=files,
         verify=not INSECURE,
     )
+    # print(res.request.url)
```
Comment (yaakovi) [inline Integrations/FidelisElevateNetwork/FidelisElevateNetwork.py]: remove this

File: Integrations/FidelisElevateNetwork/FidelisElevateNetwork.py [hunk 4]
```diff
@@ -675,6 +677,79 @@ def list_metadata():
...
+    """
+    alert_id = demisto.args().get('alert_id')
+    file_name = request_dpath(alert_id)
+    decoded_file_name = urllib.unquote(file_name)
+    results = download_malware_file_request(alert_id)
+
+    demisto.results(fileResult(
+        decoded_file_name + '.zip',
+        results,
+        file_type=entryTypes['entryInfoFile']))
```
Comment (yaakovi) [inline Integrations/FidelisElevateNetwork/FidelisElevateNetwork.py]: should malware file be downloaded as "info file" and not as "file"?
`InfoFile` is for reports and logs, `File` is for incident related files

File: Integrations/FidelisElevateNetwork/FidelisElevateNetwork.py [hunk 5]
```diff
@@ -675,6 +677,79 @@ def list_metadata():
...
+    """
+    Download PCAP from an alert
+    """
+    alert_id = demisto.args().get('alert_id')
+
+    results = download_pcap_request(alert_id)
+    demisto.results(fileResult(
+        'Alert ID_' + alert_id + '.pcap',
+        results,
+        file_type=entryTypes['entryInfoFile']))
```
Comment (yaakovi) [inline Integrations/FidelisElevateNetwork/FidelisElevateNetwork.py]: same for here

File: TestPlaybooks/playbook-Fidelis-Test.yml [hunk 6]
```diff
@@ -205,11 +214,7 @@ tasks:
       - "7"
     scriptarguments:
       alert_id:
-        complex:
-          root: Fidelis
-          accessor: Alert.ID
-          transformers:
-          - operator: GetFirstObject
+        simple: "2"
```
Comment (yaakovi) [inline TestPlaybooks/playbook-Fidelis-Test.yml]: if i recall correctly - alerts are deleted after a while, better to not use const ID

File: TestPlaybooks/playbook-Fidelis-Test.yml [hunk 7]
```diff
@@ -116,8 +123,7 @@ tasks:
       start_time:
         simple: 2019-01-09T07:50:48Z
       threat_score: {}
-      time_frame:
-        simple: Custom
+      time_frame: {}
```
Comment (yaakovi) [inline TestPlaybooks/playbook-Fidelis-Test.yml]: if it's not custom, then start_time and end_time are irrelevant

**Comment by kirbles19** [comment]
@Shellyber .yml and .md all done.

**Pull request merged**